android/src/app/organicmaps/bookmarks/Holders.java

```diff
@@ -275,6 +276,19 @@ void setSize(@PluralsRes int phrase, int size)
       mSize.setText(mSize.getResources().getQuantityString(phrase, size, size));
     }
 
+    void setZeroSize()
+    {
+      setSize(R.plurals.objects, 0);
+    }
+
+    void setDetailedSize(int tracks, int bookmarks)
+    {
+      Resources resources = mSize.getResources();
+      String tracksSize = resources.getQuantityString(R.plurals.tracks, tracks, tracks);
```

Comment:
Why the 3rd param is needed here and below?

Reply:
Quantity should be passed twice if it should also be displayed.
https://developer.android.com/guide/topics/resources/string-resource.html#Plurals

Reply:
Would API 24's MessageFormat help here? Is it backported in some Android Support library for API 21-23?

Reply:
It seems, it is not.
https://developer.android.com/topic/libraries/support-library/features
https://developer.android.com/topic/libraries/support-library/packages

It would be great to use it.